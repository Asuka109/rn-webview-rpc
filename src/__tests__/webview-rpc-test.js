/* global jest, test, expect */

import React from 'react';
import renderer from 'react-test-renderer';

import WebViewRpc from '../';


jest.mock('WebView', () => 'WebView');

test('renders correctly', () => {
  const tree = renderer.create(<WebViewRpc />).toJSON();
  expect(tree).toMatchSnapshot();
});
