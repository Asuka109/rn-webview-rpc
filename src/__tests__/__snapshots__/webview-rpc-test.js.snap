// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<WebView
  injectedJavaScript="
        var script = document.createElement(\\"script\\");
        script.src = \\"https://cdn.jsdelivr.net/npm/rn-webview-rpc\\";
        document.head.appendChild(script);
        
        "
  onMessage={[Function]}
/>
`;
